deployments/quick-start/install.sh

```
@@ -17,8 +17,8 @@ set -euo pipefail
 # Configuration
 CLUSTER_NAME="amp-local"
 CLUSTER_CONTEXT="k3d-${CLUSTER_NAME}"
-OPENCHOREO_VERSION="0.7.0"
-OC_RELEASE="release-v0.7"
+OPENCHOREO_VERSION="0.9.0"
+OC_RELEASE="release-v0.9"
```

Shellcheck (0.11.0)

[warning] OC_RELEASE appears unused. Verify use (or export if used externally).

(SC2034)